auto_py_to_exe/web/js/i18n.js

```diff
@@ -781,6 +819,9 @@ const translationMap = {
                 zh_tw: '使用單檔案模式的附加檔案時要小心;\n' +
                     '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">read this</a>\n' +
                     '並更新您的代碼以使用 PyInstaller。',
+                ua: 'Будьте уважні під час використання додаткових файлів в режимі onefile;' +
+                '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">Прочитайте це</a>\n' +
+                'і оновіть свій код для подальшого використання PyInstaller.',

```

Comment:
Just a small indentation fix: 

```suggestion
                ua: 'Будьте уважні під час використання додаткових файлів в режимі onefile;' +
                    '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">Прочитайте це</a>\n' +
                    'і оновіть свій код для подальшого використання PyInstaller.',

```

Reply:
Yep, just a second